cli/src/psp-utils/constants.ts

```diff
@@ -1,3 +1,7 @@
 export const PSP_TEMPLATE_TAG = "v0.1.1";
 export const PROGRAM_TAG = "v0.3.2";
 export const MACRO_CIRCOM_TAG = "v0.1.5";
+export const ZK_JS_VERSION = "file:../light-zk.js"; // "0.3.2-alpha.13";
+export const PROVER_JS_VERSION = "file:../light-prover.js"; // "0.1.0-alpha.1";
```

Comment:
what do the inline comments indicate? can they be removed? will we have to switch back to relative imports?
```suggestion
export const ZK_JS_VERSION = "file:../light-zk.js";
export const PROVER_JS_VERSION = "file:../light-prover.js";
```